### ISSUE TITLE
dev: support keystore

### ISSUE THREAD

**Issue by EvolveArt** [software-mansion/starknet.py#1368]
Right now, afaik it's not possible to create a signer using a keystore path that would be great to improve security of backend services using starknet.py.

Happy to provide a PR if necessary.

## References
https://github.com/xJonathanLEI/starknet-rs/blob/016f3ee3293c8355f8a7cb64777adea6119493b9/starknet-signers/src/key_pair.rs#L60

**Comment by ddoktorski** [comment]
Hi @EvolveArt 

Would it be sufficient for your use case to add a static method `from_keystore` to [KeyPair class](https://github.com/software-mansion/starknet.py/blob/8ce70350704bfe69d75c11b99d6dbe42be5cfa55/starknet_py/net/signer/stark_curve_signer.py#L14)?

**Comment by EvolveArt** [comment]
> Hi @EvolveArt
> 
> Would it be sufficient for your use case to add a static method `from_keystore` to [KeyPair class](https://github.com/software-mansion/starknet.py/blob/8ce70350704bfe69d75c11b99d6dbe42be5cfa55/starknet_py/net/signer/stark_curve_signer.py#L14)?

yes that would work, it would need to be compatible with starkli keystore ofc

**Comment by ddoktorski** [comment]
Sounds good and I agree that it does make sense to add it. Unfortunately, we're unable to prioritize it at the moment due to our current workload. However, if you could provide a PR, we would definitely be able to allocate some time to thoroughly review it.

**Comment by EvolveArt** [comment]
> Sounds good and I agree that it does make sense to add it. Unfortunately, we're unable to prioritize it at the moment due to our current workload. However, if you could provide a PR, we would definitely be able to allocate some time to thoroughly review it.

On it then, thanks!
